### DISCUSSION TITLE
Dear ross-g, I need your help with HOI4 skeleton in blender

### DISCUSSION THREAD

**Discussion by PAIMON6061** [ross-g/io_pdx_mesh#89] [Q&A] [answered]
In my blender , the skeletons are like this , they are not connected together , I don't know is it correct ? My problem is that I can't match my new mesh with the HOI4 skeleton , please help me ,  thank you very much !
![{6E540575-5B11-43b1-9849-99885FEC47D9}](https://github.com/ross-g/io_pdx_mesh/assets/46888592/54e35653-0bf5-4ce5-ba9f-bfc1d70f29f7)

And this what I can see in Maya , but I don't know Maya 
![{12493261-1D9F-4c4a-84D2-1C8594D2EBB7}](https://github.com/ross-g/io_pdx_mesh/assets/46888592/a9db095b-d8a3-4dfe-9f00-882597d56dfb)

**Comment by ross-g** [comment, marked as answer]
Yes this is normal, Paradox create their models in Maya and the bones are setup to look "correct" in Maya. They look different in Blender because bones in Blender are not the same as in Maya and the two systems aren't compatible in some ways.

If you import a HOI4 model and animation it should look correct in Blender while moving and you will still be able to create new models and animation for this skeleton. But the bones themselves will not look as natural as in Maya.

**Reply by PAIMON6061** [reply]
OK, thanks a lot! Seems like I should work hard to learn Maya btw, lol
